### PR TITLE
fix for bsc#919980

### PR DESCRIPTION
**Pull request by k0da**

### CODE REVIEW

File: test/storage_boot_on_raid1.rb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  end
+end
+
+Yast.import "Arch"
+Yast.import "Partitions"
+
+describe "CustomPartLib#CheckOkMount" do
+
+
+  it "/boot is only possible with RAID1 on PowerPC" do
```
Comment (aschnell) [inline test/storage_boot_on_raid1.rb]: The test does not test that, only that RAID1 works.

**Comment by aschnell** [comment]
REQ: Changelog is missing.

**Comment by aschnell** [comment]
LGTM

**Comment by jreidinger** [comment]
@k0da please make it mergable ( merge upstream master to your branch ) and I will merge it afterwards, thanks

**Pull request closed without merging**